docs/source/config-options.md

```diff
@@ -0,0 +1,92 @@
+## KernelGatewayApp configuration options
+
+Run `jupyter kernelgateway --help-all` after installation to see the
+full set of server options. A snapshot of this help appears below:
+
+```
+KernelGatewayApp options
+------------------------
+--KernelGatewayApp.allow_credentials=<Unicode>
+    Default: ''
+    Sets the Access-Control-Allow-Credentials header. (KG_ALLOW_CREDENTIALS env
+    var)
+--KernelGatewayApp.allow_headers=<Unicode>
+    Default: ''
+    Sets the Access-Control-Allow-Headers header. (KG_ALLOW_HEADERS env var)
+--KernelGatewayApp.allow_methods=<Unicode>
```

Comment:
Sure. We do that on other repos. For now let's leave as is and if time permits I will migrate to the autogenerate.